packages/react/src/components/Forms/Select/mono-common.tsx

```diff
@@ -135,30 +138,38 @@ export const SelectMonoAux = ({
               <div className="c__select__inner">
                 <div className="c__select__inner__value">{children}</div>
                 <div className="c__select__inner__actions">
-                  {clearable && !disabled && downshiftReturn.selectedItem && (
-                    <>
-                      <Button
-                        color="tertiary-text"
-                        size="nano"
-                        aria-label={t(
-                          "components.forms.select.clear_button_aria_label",
-                        )}
-                        className="c__select__inner__actions__clear"
-                        onClick={(e) => {
-                          downshiftReturn.selectItem(null);
-                          e.stopPropagation();
-                        }}
-                        icon={<span className="material-icons">close</span>}
-                        type="button"
-                      />
-                      <div className="c__select__inner__actions__separator" />
-                    </>
-                  )}
+                  {!props.isLoading &&
+                    clearable &&
+                    !disabled &&
+                    downshiftReturn.selectedItem && (
```

Comment:
This condition starts to be complexed. In order to explicit it we should store it into a well named variable `canBeReset` ?